tools/rna_tools/rnasnp/rnasnp.xml

```diff
@@ -37,8 +37,8 @@
         --edist=$edist
         --boltzmannPreFactor=$boltzmannPreFactor
         --winsizeFold=$mode.winsizeFold
-        > $outfile
-
+        > '$outfile'
+        ; grep "SNP" '$outfile' > /dev/null 
```

Comment:
I guess the problem is acutally the stdio that is already included, didn't saw this before. So the sdtio needs to be changed or removed ... this should fix it.